src/clj/rems/application/bonafide_bot.clj

```diff
@@ -0,0 +1,46 @@
+(ns rems.application.bonafide-bot
+  "A bot that enables workflows where a user can ask another user to vouch
+   for their bona fide researcher status."
+  (:require [clj-time.core :as time]
+            [rems.common.application-util :as application-util]
+            [rems.db.applications :as applications]
+            [rems.db.users :as users]))
+
+(def bot-userid "bonafide-bot")
+
+(defn- find-email-address [application]
```

Comment:
Should we add some protection about having just one email address? Or just use `find-first` to be more explicit about which to return?

Reply:
could be an assertion too I guess, but it's a bit impolite if it fails only when the bot tries to do something, not when creating the workflow

Reply:
added a test

Reply:
Yeah ideally there would be some collaboration with the form editor or you would pick the email field in the workflow/bot settings. Or this could be a report of its own, data validation sort of things. We don't have to do it now as long as the docs match the exact behavior.

Reply:
We could use field ids for this btw, fetching the unique field with id "referrer-email" or something?

Of course if there are multiple forms there might be multiple fields with the same field id, but we could use the first form.